lib/js/components/Well/Well.stories.ts

```diff
@@ -0,0 +1,43 @@
+import { Args, ArgTypes, Meta, StoryFn } from '@storybook/vue';
+import Well from './Well.vue';
+import { WELL_SIZES } from './Well.consts';
+
+export default {
+	title: 'Components/Well',
+	component: Well,
+} as Meta<typeof Well>;
+
+const StoryTemplate: StoryFn<typeof Well> = (argTypes) => ({
+	components: { Well },
+	props: Object.keys(argTypes),
+	template: '<well :size="size"><div #default v-html="content" /></well>',
```

Comment:
```suggestion
	template: '<well :size="size"><div v-html="content" /></well>',
```
We usually don't put name for default slot

Reply:
https://github.com/bethinkpl/design-system/pull/284/commits/960901a37711277a089fd4add7f19b67df5019c2